File: TrekOrganizer/TrekOrganizer/Models/Trek.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TrekOrganizer.Models
{
    public class Trek
    {
        public int Id { get; set; }

        public string Location { get; set; }

        public string Description { get; set; }

        public string Category { get; set; }

        public string StartDate { get; set; }

        public string EndDate { get; set; }

        public string ImageURL { get; set; }

        public int Likes { get; set; }

        public string Organizer { get; set; }
    }
}

File: trek-organizer-client/src/components/Treks/Trek/Trek.js
import { Link } from 'react-router-dom';

const Trek = (
    { trek }
) => {
    return (
        <div className="col-md-9 col-sm-8">
            <div className="post_block">
                <div className="col-md-12">
                    <div className="col-md-5">
                        <img src={trek.imageUrl} alt="" />
                    </div>
                    <div className="col-md-7">
                        <h2>{trek.location}</h2>

                        <p>{trek.description}</p>
                        <div className="catageory">
                            <ul>
                                <li><Link to="#">StartDate: {trek.startDate}</Link></li>
                                <li><Link to="#">EndDate: {trek.endDate}</Link></li>
                                <li><Link to="#">Organizer: {trek.organizerName}</Link></li>
                                {/* <li><Link to="#">3 Comments</Link></li> */}
                            </ul>
                            <div className="clearfix"></div>
                        </div>

                        <Link to={`/treks/${trek.id}`} className="read_more">Read More</Link>
                    </div>
                </div>
                <div className="clearfix"></div>
            </div>
        </div>
    )
}

export default Trek;